mapclassify/classifiers.py

```diff
@@ -93,7 +93,11 @@ def _format_intervals(mc, fmt="{:.0f}"):
     For some classifiers, it is possible that the upper bound of the first interval is less than the minimum value of the attribute that is being classified. In these cases `lower_open=True` and the lower bound of the interval is set to `np.NINF`.
     """
 
+
     lowest = mc.y.min()
+    if hasattr(mc, 'lowest'):
+        if mc.lowest:
```

Comment:
```suggestion
        if mc.lowest is not None:
```

Just to ensure we don't forget about this. We need to be explicit to ensure it works for 0.